pyproject.toml

```
@@ -23,7 +23,7 @@ chardet = ">=4,<6"
 jq = "^1.2.1"
 pandas = ">=1.5.3,<3.0.0"
 paramiko = ">=2.9.2,<4.0.0"
-pydantic = "^1.9.0"
+pydantic = ">=1.9,<3.0"
```

shouldn't we set the version to 2.0?
```suggestion
pydantic = ">=2.0,<3.0"
```

---

Thanks for pointing this out @jGundermann !

As suggested by @PrettyWood , it could be great to support both !
That's why I made this : https://github.com/ToucanToco/peakina/pull/509/commits/3d3fa2fbd46ad49a06ab50c79a0d72d25935ddf9

And I setUpped the CI to check on py3.10/3.11 with both combinations pydantic 1/2 too !
advantages : more confident on our delivery
disadvantages : long CI

Note: this is temporary, the day we will only support pydantic V2, we could fix that !